core/tests/test_utils.py

```diff
@@ -1287,12 +1342,15 @@ def run(self, result=None):
         memory_cache_services_stub.flush_cache()
 
         with contextlib2.ExitStack() as stack:
-            # Using types.MethodType is necessary because this is a classmethod
```

Comment:
Hm, interesting. Worth updating the documentation for self.swap()?

Reply:
This is already mentioned in the `swap` docstring :slightly_smiling_face:

Reply:
Done.